mqcloudv1/mqcloud_v1_examples_test.go

```diff
@@ -740,11 +746,105 @@ var _ = Describe(`MqcloudV1 Examples Tests`, func() {
 			Expect(response.StatusCode).To(Equal(200))
 			Expect(channelsDetails).ToNot(BeNil())
 		})
+		It(`CreateVirtualPrivateEndpointGateway request example`, func() {
+			fmt.Println("\nCreateVirtualPrivateEndpointGateway() result:")
+			// begin-create_virtual_private_endpoint_gateway
+
+			createVirtualPrivateEndpointGatewayOptions := mqcloudService.NewCreateVirtualPrivateEndpointGatewayOptions(
+				config["MQCLOUD_SERVICE_INSTANCE_CAPACITY_GUID"],
```

Comment:
```suggestion
				config["SERVICE_INSTANCE_CAPACITY_GUID"],
```
Seeing usage of both `SERVICE_INSTANCE_CAPACITY_GUID` and `MQCLOUD_SERVICE_INSTANCE_CAPACITY_GUID` vars

Just to maintain consistency while exporting env vars.